RealmSwift/Tests/SwiftUITests.swift

```diff
@@ -286,6 +286,28 @@ class SwiftUITests: TestCase {
         state.projectedValue.remove(object)
         XCTAssertEqual(state.wrappedValue.count, 0)
     }
+    func testSwiftQuerySyntax() throws {
+        let obj = SwiftUIObject()
+        obj.str = "foo"
+        let realm = inMemoryRealm(inMemoryIdentifier)
+        try! realm.write {
+            realm.add(obj)
+        }
+
+        var filteredResults = ObservedResults(SwiftUIObject.self,
+                                              configuration: inMemoryRealm(inMemoryIdentifier).configuration,
+                                              where: { $0.str == "foo" },
+                                              sortDescriptor: SortDescriptor.init(keyPath: \SwiftUIObject.str, ascending: true))
```

Comment:
This sort descriptor doesn't look like it is adding any extra value to the test.